chamber/utils/transaction.py

```diff
@@ -43,9 +47,11 @@ class TransactionSignalsContext:
     and executed only once.
     """
 
-    def __init__(self):
-        self._unique_callable = OrderedDict()
+    def __init__(self, sid, in_atomic_block):
```

Comment:
What is `sid`?

Reply:
`session_id` would be a little more explanatory.

Reply:
I don't know I am using name get from django. So it should be same